packages/swapper/src/swappers/thorchain/utils/getLimit/getLimit.test.ts

```diff
@@ -51,7 +51,7 @@ describe('getLimit', () => {
       buyAssetTradeFeeUsd: '6.2049517907881932',
     }
     const limit = await getLimit(getLimitArgs)
-    expect(limit).toBe('580583')
+    expect(limit).toBe('577495')
```

Comment:
These are slightly less than they used to be due to the addition of `ALLOWABLE_MARKET_MOVEMENT`.